README.md

```diff
@@ -171,7 +171,7 @@ kubectl --namespace kube-system patch deploy tiller-deploy \
 
 The project is released under version 2.0 of the [Apache license](http://www.apache.org/licenses/LICENSE-2.0).
 
-[helm-hub]: https://hub.helm.sh/charts/newrelic
+[helm-hub]: https://artifacthub.io/packages/search?ts_query_web=newrelic
```

Comment:
Just a nit to narrow the search to this repo
```suggestion
[helm-hub]: https://artifacthub.io/packages/search?repo=newrelic&ts_query_web=newrelic
```

Reply:
I would say to use either the repo filter or the search filter, as having both seems redundant.
```suggestion
[helm-hub]: https://artifacthub.io/packages/search?repo=newrelic
```